modules/stochastic_tools/include/state_sim/common/StateSimBitset.h

```diff
@@ -0,0 +1,61 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMBITSET_H
+#define STATESIMBITSET_H
+
+#include <vector>
+#include <stdexcept>
+
+/**
+ * StateSimBitset is a bitset library to handle fast ID lookup for StateSim objects, only needed bitsest functions were defined.
+ */
+class StateSimBitset
+{
+protected:
+  std::vector<unsigned int> _m_array;
+  int _m_length;
+
+
+private:
+  void checkOperand(const StateSimBitset & in_set) const;
+
+
+public:
+  /**
+   * Copy construter.
+   * @in_set - item to copy
+   */
+  StateSimBitset(const StateSimBitset& in_set);
```

Comment:
I'm not sure I understand what is wrong.  Is it suppose to be 
StateSimBitset(const StateSimBitset & in_set);